drop/download_data.sh

```diff
@@ -3,15 +3,19 @@ set -e
 
 # get data
 resource_url="https://www.cmm.in.tum.de/public/paper/drop_analysis/resource.tar.gz"
-wget -Nc $resource_url
-tar -zxvf resource.tar.gz
-rm -rf Data
-mv resource Data
+wget -nc $resource_url
```

Comment:
```
tmpresource="$(dirname $(tempfile))/drop_resource.tar.gz"
wget -nc -O$tmpresource $resource_url
```
We should use this so we can download it once and keep it in temp.

Reply:
fixed in 9640e3f